src/pages/home/report/EmojiPickerMenu/index.native.js

```diff
@@ -68,6 +68,9 @@ class EmojiPickerMenu extends Component {
                     renderItem={this.renderItem}
                     keyExtractor={item => (`emoji_picker_${item.code}`)}
                     numColumns={this.numColumns}
+                    removeClippedSubviews
+                    maxToRenderPerBatch={this.numColumns}
+                    windowSize={3}
```

Comment:
same as above

Reply:
I was able to confirm locally that removing these line fixes [a regression](https://github.com/Expensify/Expensify.cash/issues/2660)